src/Components/StatusPills/ColorPills.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import { IColorPillsProps } from './StatusPills.types';
+import { getPillStyles, getStyles } from './StatusPills.styles';
+import { IStackTokens, Stack } from '@fluentui/react';
+import { useId } from '@fluentui/react-hooks';
+
+export const ColorPills: React.FC<IColorPillsProps> = (props) => {
+    const { visualColorings, width } = props;
+    const stackTokens: IStackTokens = { childrenGap: 2 };
+    const styles = getStyles(width);
+    const alignment = width === 'wide' ? 'center' : 'start';
+    const pillId = useId('cb-visual-rule-pills');
+    const [trimmedVisualColorings, setTrimmedVisualColorings] = useState(
+        visualColorings && visualColorings.length > 3
+            ? visualColorings.slice(0, 3)
+            : visualColorings
+    );
+    const [extraVisualColoringCount, setExtraVisualColoringCount] = useState(
+        visualColorings && visualColorings.length > 3
+            ? visualColorings.length - 3
+            : 0
+    );
+
+    useEffect(() => {
+        setTrimmedVisualColorings(
+            visualColorings && visualColorings.length > 3
+                ? visualColorings.slice(0, 3)
+                : visualColorings
+        );
+        setExtraVisualColoringCount(
+            visualColorings && visualColorings.length > 3
+                ? visualColorings.length - 3
+                : 0
+        );
+    }, [visualColorings]);
+
+    return (
+        <Stack
+            tokens={stackTokens}
+            horizontal={true}
+            horizontalAlign={alignment}
+            verticalAlign={'center'}
+            styles={{ root: styles.root }}
+        >
+            {trimmedVisualColorings.map((vc, idx) => {
+                const { color } = vc;
+                const pillStyles = getPillStyles(color);
+
+                return (
+                    <div
+                        key={`${pillId}-${idx}`}
```

Comment:
what do we need to do here for accessibility (mainly screen readers)? I don't know what the guidance is since this is a color. It's not like we can read the hex code out. 😟

Reply:
aria-hide this?

Reply:
should we set it to aria-hidden? I don't know whether screen readers will see this since nothing actually shows up in teh UI?

Reply:
Now that I think about it, this has no content, if it had a title or some aria label it would be mentioned by screen readers, but a div with a certain background should be ignored. The colors that can be used for visual rules are limited, I could do a map of color to aria label. Sounds a bit overkill